src/index.ts

```diff
@@ -119,6 +146,23 @@ function parseLineHeight(value: string) {
 	const parsed = parseFloat(value);
 	if (parsed.toString() === value) {
 		return parsed;
+	} else {
+		const match = /^(\+|-)?(\.)?/.exec(value) as RegExpMatchArray;
+		let val: string = value;
+		if (match[1] === '+') {
+			val = val.substring(1);
+		}
+		if (match[2] === '.') {
+				// NOTE although not specifically prohibited, we do not consider negative numbers for line-height valid:
```

Comment:
Were all these comments left here on purpose? Were these issues ever resolved?

Reply:
yes, I left this comment on purpose, although I will happily remove it:

The underlying reason for the comment was that I am not really sure why there is a conversion from `string` to  `number` at this point.

I did not want to touch the `line-height` parsing code at all, but the test framework requires the conversion of the `string` values to `number` (otherwise the extended number tests could not have been added, or would have needed to exclude tests where `line-height` was part of the test input).

The comment is wrong in that really [negative numbers are not allowed for `line-height`](https://drafts.csswg.org/css2/visudet.html#propdef-line-height), but as I said, I did not want to mess around too much with this code as I was not really sure what the intention was here:
why convert to numbers? and why omit the units?